src/containers/SecretsModal/SecretsModal.js

```diff
@@ -78,8 +78,13 @@ export /* istanbul ignore next */ class SecretsModal extends Component {
     if (webSocketConnected && prevWebSocketConnected === false) {
       this.props.fetchServiceAccounts();
     }
+    this.scrollToBottom();
```

Comment:
Wouldn't we only want to do this if a new annotation was added? It could be quite disruptive if it randomly scrolled down on some unrelated update.

Reply:
yeah I realised this after I put in the PR - working on it only doing it when an annotation is added now.